lib/search_click_tracking/service.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require 'uri'
+require 'common/client/base'
+require 'common/client/concerns/monitoring'
+require 'common/client/errors'
+require 'search_click_tracking/response'
+require 'search_click_tracking/configuration'
+
+module SearchClickTracking
+  # This class builds a wrapper around Search.gov web results API. Creating a new instance of class
+  # will and calling #results will return a ResultsResponse upon success or an exception upon failure.
+  #
+  # @see https://search.usa.gov/sites/7378/api_instructions
+  #
+  class Service < Common::Client::Base
+    include Common::Client::Concerns::Monitoring
+
+    STATSD_KEY_PREFIX = 'api.search_click_tracking'
+
+    configuration SearchClickTracking::Configuration
+
+    attr_reader :url
+    attr_reader :query
+    attr_reader :position
+    attr_reader :client_ip
+    attr_reader :user_agent
+
+    def initialize(url, query, position, client_ip, user_agent)
+      @url = url
+      @query = query
+      @position = position
+      @client_ip = client_ip
+      @user_agent = user_agent
+    end
+
+    # POSTs click tracking query param data to search.gov
+    #
+    def track_click
+      Faraday.post(url_with_params, '')
```

Comment:
It doesn't look like you're utilizing the monitoring concern/with_monitoring here to increment the statsd metric